### ISSUE TITLE
TypeError: this._cache.unshift is not a function

### ISSUE THREAD

**Issue by felipe19930** [JustArchiNET/ASF-ui#161]
Hello,

I'm getting the following javascript error whenever I try to send ASF commands in the "Commands" page:

```js
Commands.vue:44 Uncaught (in promise) TypeError: this._cache.unshift is not a function
    at CommandsCache.add (Commands.vue:44)
    at VueComponent.sendCommand (Commands.vue:199)
    at _c.on.keydown (Commands.vue?6d25:59)
    at invoker (vue.runtime.esm.js:2019)
    at HTMLInputElement.fn._withTask.fn._withTask (vue.runtime.esm.js:1822)
```

If I clear the command history by running `localStorage.removeItem("asf-ui:command-history")` in the developer console and then refresh the page, sending commands start to work again. But once I close the tab or refresh the page, the error shows up again:

https://gfycat.com/IckyShamefulAllosaurus

**Comment by felipe19930** [comment]
Forgot to mention that I'm on commit 59b1724

**Comment by Aareksio** [comment]
Since you are on one of latest commits, mind turning `logging` on and providing the log?
Toggle it on, reproduce the error and either copy the log or enable reporting.

Something is probably broken regarding cache recovery

**Comment by Aareksio** [comment]
Fixed in a15a201eb9b9df365cadbe85eec30b65d8c59a49

**Comment by Aareksio** [comment]
Thanks for testing latest release!
